src/components/Counter/Counter.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export function Counter({ count, addOne, add100, increase, reset }) {
+  return (
+    <>
+      <h1>{`Count: ${count}`}</h1>
+      <div className="buttons-container">
+        <Button
+          variant="outline-primary"
+          onClick={() => addOne()}
+        >
+          + 1
+        </Button>
+        <Button
+          variant="outline-primary"
+          onClick={() => add100()}
+        >
+          + 100
+        </Button>
+        <Button
+          variant="outline-primary"
+          onClick={() => increase()}
+        >
+          Increase
+        </Button>
+        <Button
+          variant="outline-danger"
+          onClick={() => reset()}
```

Comment:
```suggestion
          onClick={reset}
```
если ты не передаешь параметры в функцию, можно записывать так

Reply:
поменяй везде